lib/Core/Executor.cpp

```diff
@@ -1249,40 +1250,46 @@ void Executor::executeCall(ExecutionState &state,
           // Alignment requirements for scalar types is the same as their size
```

Comment:
This comment doesn't seem to match the implementation. The comment seems to suggest that ``size = llvm::roundUpToAlignment(size, size)``.